docs/pages/custom-builds/get-started.mdx

```diff
@@ -10,7 +10,11 @@ import ImageSpotlight from '~/components/plugins/ImageSpotlight';
 import { BoxLink } from '~/ui/components/BoxLink';
 import { GithubIcon } from '@expo/styleguide-icons';
 
-> **Warning** All content in this document is experimental, under active development, and likely to change. Use at your own risk. If you're looking to execute custom code before or after certain steps in the build process, but don't need to actually modify any of the steps, you may want to use the stable [build lifecycle hooks API](/build-reference/npm-hooks/).
+> **Warning** This functionality is in preview, under active development, and it can be subject to changes.
+
+{/* TODO: link to the blog post */}
+
+> **Warning** Current limitations of custom builds are highlighted in the [changelog](https://expo.dev/changelog/2023/08-10-custom-build-jobs) and include not adapted pricing, limitations regarding uploading artifacts, lack of reusable steps for caching dependencies, no support for classic updates, and not ready GitHub integration for custom builds.
 
 Custom workflows allow running tests on the cloud, adding instructions for native platforms, resigning a build to share with your team or multiple test devices, and so on. With EAS Build, you can create build configs to customize the process by adding your workflows. It helps saving time especially when working in teams.
```

Comment:
```suggestion
Custom builds allow running tests on the cloud, adding instructions for native platforms, resigning a build to share with your team or multiple test devices, and so on. With EAS Build, you can create build configs to customize the process, so that you're not limited to just generating Android and iOS applications and you can do anything that you might want to do in a CI environment.
```